webpay/base/dev_messages.py

```diff
@@ -15,6 +15,7 @@
 BAD_REQUEST = 'BAD_REQUEST'
 BAD_SIM_RESULT = 'BAD_SIM_RESULT'
 BANGO_ERROR = 'BANGO_ERROR'
+BUYER_NOT_CONFIGURED = 'BUYER_NOT_CONFIGURED'
```

Comment:
You can move the informative, developer-centric explanations from [this patch](https://github.com/mozilla/spartacus/pull/219/files#diff-22780b0eafac14271cb806bb1c518d79R12) over to the [legend down below](https://github.com/andymckay/webpay/blob/1140615-reboot/webpay/base/dev_messages.py#L133). The error explanations in this legend are for developers.

The concept is this: when a user hits an error in the buy flow we show 1) a prominent message that helps them understand why they cannot purchase the item and 2) an error code that can be used for diagnostics. The user won't care about the error code because it won't help them. However, a developer may be testing their own app and when they see an error code, they have the opportunity to use our [legend API](https://webpay.readthedocs.org/en/latest/services.html#error-legend) to get more information about the error code.